package.json

```diff
@@ -31,7 +31,7 @@
     "@angular/platform-browser": "~9.1.2",
     "@angular/platform-browser-dynamic": "~9.1.2",
     "@angular/router": "~9.1.2",
-    "@knora/api": "1.0.0-rc.0",
+    "@knora/api": "1.0.0-rc.1",
```

Comment:
Could you also commit the change in `package-lock.json`? It still says 1.0.0-rc.0.